sirepo/srunit.py

```diff
@@ -317,6 +325,27 @@ def sr_get_root(self, sim_type=None, **kwargs):
             **kwargs,
         )
 
+    def sr_iter_sleep(self, kind, op_desc):
```

Comment:
This no longer needs to be 'sr_', because it no longer inherits from flask.TestClient.

Reply:
d0bb36461